libraries/Native_Ethernet/src/utility/stm32_eth.h

```diff
@@ -58,21 +61,28 @@ typedef enum
   TCP_CLOSING,
 }tcp_client_states;
 
-struct udp_struct {
-  struct udp_pcb *pcb;
+/* Struct to store received data */
+struct pbuf_data {
   struct pbuf *p;     // the packet buffer that was received
   uint16_t available; // number of data
+};
+
+/* UDP structure */
+struct udp_struct {
+  struct udp_pcb *pcb; /* pointer on the current udp_pcb */
+  struct pbuf_data data;
```

Comment:
not sure that this is related to this commit "rename library ..."
shouldn't this be part of previous one ?